src/test/java/org/mule/tools/devkit/sonar/checks/maven/DevkitLatestVersionCheckTest.java

```diff
@@ -0,0 +1,127 @@
+package org.mule.tools.devkit.sonar.checks.maven;
+
+
+import com.google.common.base.Splitter;
+import org.apache.maven.project.MavenProject;
+import org.codehaus.plexus.util.xml.pull.XmlPullParserException;
+import org.jetbrains.annotations.NotNull;
+import org.junit.Test;
+import org.mule.tools.devkit.sonar.utils.PomUtils;
+
+import javax.xml.stream.XMLInputFactory;
+import javax.xml.stream.XMLStreamException;
+import javax.xml.stream.XMLStreamReader;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.URL;
+import java.util.List;
+
+import static org.junit.Assert.assertThat;
+import static org.hamcrest.Matchers.is;
+public class DevkitLatestVersionCheckTest {
+
+    @Test
+    public void checkDevkitVersionIsTrue() throws IOException, XmlPullParserException, XMLStreamException {
+        final MavenProject mavenProject = PomUtils.createMavenProjectFromPomFile(new File("src/test/files/maven/devkit-latest-version/devkit-is-latest-version"));
+        Version currentVersion = new Version(mavenProject.getModel().getParent().getVersion());
+        Version latestVersion = getLatestVersion();
+        assertThat(currentVersion.compareTo(latestVersion), is(0));
+    }
+
+    @Test
+    public void checkDevkitVersionIsFalse() throws IOException, XmlPullParserException, XMLStreamException {
+        final MavenProject mavenProject = PomUtils.createMavenProjectFromPomFile(new File("src/test/files/maven/devkit-latest-version/devkit-is-not-latest-version"));
+        Version currentVersion = new Version(mavenProject.getModel().getParent().getVersion());
+        Version latestVersion = getLatestVersion();
+        assertThat(currentVersion.compareTo(latestVersion), is(-1));
+    }
+
+    public Version getLatestVersion() throws IOException, XMLStreamException {
```

Comment:
Thanks! Thats exactly what i wanted to know, where to extract that object and method